examples/images_contours_and_fields/barcode_demo.py

```diff
@@ -1,33 +1,40 @@
 """
-============
-Barcode Demo
-============
-
-This demo shows how to produce a one-dimensional image, or "bar code".
+=======
+Barcode
+=======
+This demo shows how to produce a bar code.
+
+The figure is calculated so that the width in pixel is a multiple of the
+number of data points to prevent interpolation artifacts. Additionally, the
+axes is defined to span the whole figure and axis are turned off.
```

Comment:
Otherwise, the verb plurality is confusing...
```suggestion
``Axes`` is defined to span the whole figure and all ``Axis`` are turned off.
```
(feel free to link instead.)